backend/routes/api/users.js

```diff
@@ -26,28 +27,119 @@ const validateSignup = [
       .exists({ checkFalsy: true })
       .isLength({ min: 6 })
       .withMessage('Password must be 6 characters or more.'),
+    check('firstName')
+      .exists({ checkFalsy: true })
+      .withMessage('First Name is required'),
+    check('lastName')
+      .exists({ checkFalsy: true })
+      .withMessage('Last Name is required'),
     handleValidationErrors
   ];
 
+// Validate Login
+const validateLogin = [
+  check('credential')
+    .exists({ checkFalsy: true })
+    .notEmpty()
+    .withMessage('Please provide a valid email or username.'),
+  check('password')
+    .exists({ checkFalsy: true })
+    .withMessage('Please provide a password.'),
+  handleValidationErrors
+];
+
```

Comment:
You don't need to change this, having your validators here is totally valid. Another way that could be done is to make a validators folder in your utils folder and move all of your validations into that folder so that you can just import them into your routes. Up to you, some people prefer it to be in the same file others like to have them sectioned off in their own area